docs/usage/configuration/config_documentation.md

```diff
@@ -230,6 +230,21 @@ Example configuration:
 serve_server_wellknown: true
 ```
 ---
+Config option: `custom_well_known`
```

Comment:
Bit of a nitpick while passing by and skimming through this PR, I would rename this config option into something like `extra_well_known_content`. As it stands it's not clear to me from the option's name that this option is not replacing the content that's automatically generated.